Assets/Mirror/Runtime/Transport/Websocket/WebsocketTransport.cs

```diff
@@ -6,7 +6,7 @@ namespace Mirror.Websocket
     public class WebsocketTransport : Transport
     {
 
-        protected Client client = new Client();
+        protected Client client;
         protected Server server = new Server();
```

Comment:
what about server max message size? that's the one that matters for allocation attacks